### DISCUSSION TITLE
OpenTOPAS meeting at AAPM

### DISCUSSION THREAD

**Discussion by jschuemann** [OpenTOPAS/OpenTOPAS#57] [Announcements]
Hi,
in case you missed it, we will have an user meeting at AAPM. Come by Tuesday, 6-7pm in the Headquarter Hotel, Olympic II. 

The night out starts at 6, but I am told the game won't start until 7:30 and there will be busses going back and forth, so you can still enjoy the night out after the OpenTOPAS meeting.
See you there.
Cheers
       Jan

**Comment by Wonmo** [comment]
Hi Jan, could you please provide the address? thank you for organizing!

**Reply by JoseRamosMendez** [reply]
![image](https://github.com/user-attachments/assets/3f6ae38b-40b1-48b6-ab73-16a26dc832bd)
